rllib/algorithms/bc/tf/bc_tf_rl_module.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Mapping
+
+from ray.rllib.algorithms.bc.bc_rl_module import BCRLModule
+from ray.rllib.core.models.base import ENCODER_OUT, STATE_OUT
+from ray.rllib.core.rl_module.rl_module import RLModule
+from ray.rllib.core.rl_module.tf.tf_rl_module import TfRLModule
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.nested_dict import NestedDict
+from ray.rllib.utils.framework import try_import_tf
+
+tf1, tf, _ = try_import_tf()
```

Comment:
We don't need any tf here!
Same goes for the torch version!

@simonsays1980 Please consolidate the methods of both RL Modules into one `BCBaseRLModule`.
This `BCBaseRLModule` should have the docstrings that you currently have in the torch version.
If possible, the two framework-specific RL Modules should just be:

```
class BCTfRLModule(TfRLModule, BCRLModule):
    framework: str = "tf2"
```

and

```
class BCTorchRLModule(TorchRLModule, BCRLModule):
    framework: str = "torch"
```